onetl/base/base_db_connection.py

```diff
@@ -60,19 +61,15 @@ def validate_columns(cls, connection: BaseDBConnection, columns: list[str] | Non
 
     @classmethod
     @abstractmethod
-    def validate_hwm_column(
-        cls,
-        connection: BaseDBConnection,
-        hwm_column: str | None,
-    ) -> str | None:
-        """Check if ``hwm_column`` value is valid.
+    def validate_hwm(cls, connection: BaseDBConnection, hwm: HWM) -> HWM:
+        """Check if ``hwm`` class is valid.
```

Comment:
```suggestion
        """Check if ``HWM`` class is valid.
```
nit: название класса пишется заглавными буквами